index.d.ts

```diff
@@ -0,0 +1,55 @@
+type RecordObject = import('cql-execution').RecordObject;
+type PatientObject = import('cql-execution').PatientObject;
+type DataProvider = import('cql-execution').DataProvider;
+type RetrieveDetails = import('cql-execution').RetrieveDetails;
+type Code = import('cql-execution').Code;
+type Date = import('cql-execution').Date;
+type Interval = import('cql-execution').Interval;
+
+class FHIRObject implements RecordObject {
+  constructor(json: any, typeInfo: any, modelInfo: any);
+  get(field: string): any;
+  getId(): string;
+  getCode(field: string): Code;
+  getDate(field: string): Date;
```

Comment:
Actually, this returns a `DateTime`.
```suggestion
  getDate(field: string): DateTime;
```